bebop_driver/src/bebop.cpp

```diff
@@ -85,6 +85,34 @@ namespace bebop_driver
 
 const char* Bebop::LOG_TAG = "BebopSDK";
 
+/*
+ * @brief Set device to controller port. Function should be called before ARCONTROLLER_Device_t init.
+ * @warning Dirty hack.
+ * @param device_ptr_ Pointer to discovery device.
+ * @param _port Device to controller port.
+ * @return void. 
+ */
+void setD2CPort( ARDISCOVERY_Device_t* device_ptr_, const int _d2cport )
+{
+    // Define structure from ARDISCOVERY_DEVICE_Wifi.h
```

Comment:
`libardiscovery` should be accessible here, can't we just `include` the header and use the defined data type for `ARDISCOVERY_DEVICE_WIFI_t`?